### PR TITLE
Rn entry 2023.10.0

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adding release notes for 2023.10.0
- **Is this PR related to a Monolith release?** Yes. 2023.10.0

### Resolves:

* [OKTA-654597](https://oktainc.atlassian.net/browse/OKTA-654597)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,41 @@ title: Okta Identity Engine API Products release notes 2023
...
+| ------ | ------------------------ |
+| [Permission conditions for profile attributes is GA in Preview](#permission-conditions-for-profile-attributes-is-ga-in-preview) | October 12, 2023 |
+| [Direct authentication endpoints now require channel_hint parameter](#direct-authentication-endpoints-now-require-channel_hint-parameter) | October 12, 2023 |
+| [Updates to profile enrollment policy are GA in Production](#updates-to-profile-enrollment-policy-are-ga-in-production)  | September 13, 2023  |
+| [Bugs fixed in 2023.10.0](#bugs-fixed-in-2023-10-0) | October 12, 2023 |
+
+#### Permission conditions for profile attributes is GA in Preview
+
+You can now apply conditions to the View users and their details and Edit users' profile attributes custom admin role permissions. Permission conditions help you limit the scope of a role by including or excluding admins' access to individual profile attributes. This gives you more granular control over your custom admin roles and helps meet your org’s unique security needs. See [Permission conditions](https://help.okta.com/okta_help.htm?type=oie&id=ext-permission-conditions). <!-- OKTA-586185 FF: CUSTOM_ADMIN_ROLES_CONDITIONS -->
```
[the remark this answers is not in the document]
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:22]: Match any changes to the Classic blurb as well.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,6 +6,41 @@ title: Okta Identity Engine API Products release notes 2023
...
 > Help us improve our release notes by filling out this short [survey](https://surveys.okta.com/jfe/form/SV_4VEZcIGOX0TBgkC).
 
+## October
+
+### Monthly release 2023.10.0
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Permission conditions for profile attributes is GA in Preview](#permission-conditions-for-profile-attributes-is-ga-in-preview) | October 12, 2023 |
+| [Direct authentication endpoints now require channel_hint parameter](#direct-authentication-endpoints-now-require-channel_hint-parameter) | October 12, 2023 |
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:16]: This should fix the link.
```suggestion
| [Direct authentication endpoints now require channel_hint parameter](#direct-authentication-endpoints-now-require-channel-hint-parameter) | October 12, 2023 |
```

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 3]
```diff
@@ -6,6 +6,41 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Bugs fixed in 2023.10.0
+
+* If an org had custom character restrictions on the `login` variable, POST requests to the `/idp/idx/enroll/new` endpoint with invalid `login` values returned an incorrect error message. (OKTA-631952)
+
+* When an admin configured a custom email domain using the Email Domain API, they couldn't change the autogenerated CNAME record. (OKTA-645037)
+
+* The Apps API displayed an error when admins tried to remove the Refresh Token grant type. (OKTA-630471)
+
+* An Authentication Policy rule configured with `methods`/`types` `constraints` parameters incorrectly prompted users for a security question. (OKTA-642454)
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:40]: Are these multiple params?

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md]: Getting clarification from Brian.

**Pull request merged**